EpiResponsivePicture/ResizedImage/Processors/ResizeWebDownscaleProcessor.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.Extensions.Logging;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Metadata.Profiles.Exif;
+using SixLabors.ImageSharp.Web;
+using SixLabors.ImageSharp.Web.Commands;
+using SixLabors.ImageSharp.Web.Processors;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace Forte.EpiResponsivePicture.ResizedImage.Processors
+{
+    public class ResizeWebDownscaleProcessor : IImageWebProcessor
+    {
+        public static string Width => ResizeWebProcessor.Width;
+        public static string Height => ResizeWebProcessor.Height;
+        public static string Orient => ResizeWebProcessor.Orient;
+
+        private readonly ResizeWebProcessor _processor = new();
```

Comment:
Sure, we can inject it, however it can not be registered in DI. Otherwise it will be used by ImageSharp, which we do not want. What we can do is to move object creation to `AddForteEpiResponsivePicture` method like so: `.AddProcessor(_ => new ResizeWebDownscaleProcessor(new ResizeWebProcessor()))`. I also change reference type from `ResizeWebProcessor` to `IImageWebProcessor` to be more flexible.